platform-sdk/swirlds-fchashmap/src/test/java/com/swirlds/fchashmap/FCHashMapTests.java

```diff
@@ -448,7 +448,7 @@ void insertAndRemoveQuickly() {
     @DisplayName("Null Value Test")
     void nullValueTest() {
         final FCHashMap<Integer, Integer> map = new FCHashMap<>();
-        assertThrows(IllegalArgumentException.class, () -> map.put(0, null), "map should not accept null values");
+        assertThrows(NullPointerException.class, () -> map.put(0, null), "map should not accept null values");
```

Comment:
why has that changed?

Reply:
Changed from `CommonUtils.throwArgNull` to `ArgumentUtils.throwArgNull`